Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -244,7 +264,28 @@ public void AddRow( DataGridRowInfo<TItem> row )
     /// </summary>
     /// <param name="column">Column to link with this datagrid.</param>
     public bool RemoveColumn( DataGridColumn<TItem> column )
-        => Columns.Remove( column );
+    {
+        var removed = Columns.Remove( column );
+        
+        if ( column.SortDirection != SortDirection.Default )
+        {
+            SortByColumns.Remove( column );

```

Comment:
So, wasn't this already handled before? Would it remain in the SortByColumn list?
I guess this is only really going to matter if someone plays with dynamic columns though.